invisible_cities/io/dst_io_test.py

```diff
@@ -155,4 +156,14 @@ def test_store_pandas_as_tables_raises_warning_empty_dataframe(config_tmpdir, em
     table_name = 'table_name_3'
     with tb.open_file(filename, 'w') as h5out:
         with pytest.warns(UserWarning, match='dataframe is empty'):
-            _store_pandas_as_tables(h5out, empty_dataframe, group_name, table_name)
+            store_pandas_as_tables(h5out, empty_dataframe, group_name, table_name)
+
+@given(df=lstrings_dataframe)
+def test_store_pandas_as_tables_raises_warning_long_string(config_tmpdir, df):
+    filename   = config_tmpdir + 'dataframe_to_table_exact.h5'
+    group_name = 'test_group'
+    table_name = 'table_name_3'
+    with tb.open_file(filename, 'w') as h5out:
+        with pytest.warns(UserWarning, match='dataframe contains strings longer than allowed'):
+            store_pandas_as_tables(h5out, df, group_name, table_name)
```

Comment:
I suggest that here you set explicitly the string column length in the call to `store_pandas_as_tables` to make the test more obvious.